### PR TITLE
Add changes to file

### PR DESCRIPTION
**Pull request by obed300**
In this project, I  added some functionality to my application to make it interactive. The user is able to mark task completion by selecting the corresponding checkbox (or undo it by unchecking the checkbox). The updated tasks list is stored in local storage.

### CODE REVIEW

**Review by OyePriscilla — COMMENTED** [review]
Hi @obed300 👍, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

**You have submitted a project, that has no JavaScript Linter (eslint) setup.**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by rloterh — COMMENTED** [review]
Hi Obed,
 
 While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
 
## Invalid Code Review Request
You have submitted a project, whereby ESLint is not properly setup, and hence, ESLint checks not showing on your Pull Request. Kindly ensure that **all** required linters are properly configured, and checks are passing. 

**ESLint not showing:**
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/modules/func.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+  static listlibrary (element)  {
+    const listcontainer = document.querySelector('.list-container');
+
+    const list = document.createElement('div');
+    list.classList.add('mylist');
+    const check = document.createElement('input');
+    check.type = 'checkbox';
+    // check.checked = true;
+    // check.id = 'checkbox';
```
Comment (BertrandConxy) [inline src/modules/func.js]: - [ ] Please remove the following commented codes. It is among the best practices to remove unnecessary commented codes to keep your codebase clean 😉 👍

File: src/modules/delete.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// import UI from './func.js';
```
Comment (BertrandConxy) [inline src/modules/delete.js]: - [ ] Please the following commented line of code too 😃 👍 this will help you keep your files clean 😉

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @obed300, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/func.js (hunk 1 above)
Comment (obed300) [reply src/modules/func.js]: I will do that. Thank you

File: src/modules/delete.js (hunk 2 above)
Comment (obed300) [reply src/modules/delete.js]: Thank you

**Review by prolajumokeoni — APPROVED** [review]
Hi @obed300 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀  
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**